dev-python/pyasyncore/pyasyncore-1.0.4.ebuild

```diff
@@ -17,6 +17,14 @@ HOMEPAGE="
 LICENSE="PSF-2"
 SLOT="0"
 KEYWORDS="~alpha amd64 arm arm64 ~loong ppc ppc64 ~riscv sparc x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	test? (
+		dev-python/test[${PYTHON_USEDEP}]
```

Comment:
Bit worried about the namespace of this, but I guess that would be a problem anyway.